packages/keystone/databaseAdapters/BalancingReplicaKnexAdapter/adapter.js

```
@@ -13,179 +13,260 @@
+            await Promise.all(Object.values(this._knexClients).map((k) => k.destroy()));
+          }
 
-            // Close connections gracefully
-            await this.knex.destroy()
-            await Promise.all(Object.values(this._knexClients).map(knex => knex.destroy()))
+          throw new Error(`One or more databases has non-supported versions.\n${errorDetails}`);
+        }
+    }
 
-            throw new Error(`One or more databases has non-supported versions.\n${errorDetails}`)
+    async disconnect() {
```

ESLint

[error] Missing space before function parentheses.

(@typescript-eslint/space-before-function-paren)